frontend/src/routes/static/aktuelles/+page.svelte

```diff
@@ -1,3 +1,9 @@
-<div class="p-10">
-    Place your HTML here. (News page)
+<svelte:options runes={true} />
+
```

Comment:
Looking at the issue there should be a title + desc text here:

<img width="1136" height="139" alt="image" src="https://github.com/user-attachments/assets/d9ca6b96-d552-4246-b486-41f215c7476d" />